source/integrations.txt

```diff
@@ -76,70 +71,59 @@ To learn more about Spring Data MongoDB and its features, see the `Spring Data M
 Quarkus
 -------
 
-`Quarkus <https://quarkus.io/>`__, a Java framework, aims to leverage the functionality of
-`Kubernetes <https://kubernetes.io/>`__ to create lightweight applications. Much like the
-Spring framework described in the :ref:`spring-framework` section of this page, Quarkus
-offers improved code efficiency and builds on a range of third-party libraries.
-Quarkus also supports a variety of application styles, including HTTP microservices,
-reactive applications, and serverless functions.
+The `Quarkus <https://quarkus.io/>`__ Java framework supports the development of microservices
+and serverless systems. Quarkus is native to `Kubernetes <https://kubernetes.io/>`__, a platform
+that manages the deployment of containerized applications.
+
+Quarkus' features include an interactive developer interface, live coding, and container-first application
+development that reduces the memory usage of Java applications.
 
 Using Quarkus and MongoDB
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Quarkus offers the ``quarkus-mongodb-client`` extension to connect to MongoDB. This
-extension allows you to create a Quarkus application that configures a MongoDB ``Client``
-instance and interacts with data stored in MongoDB.
+Quarkus provides a MongoDB client for MongoDB data access. For instructions on how to use
+the ``quarkus-mongodb-client`` extension to connect to MongoDB, see the `Using the MongoDB Client
+<https://quarkus.io/guides/mongodb>`__ guide in the Quarkus documentation.
 
-Additionally, you can install the ``quarkus-mongodb-panache`` extension to further
-simplify MongoDB database interactions. This extension wraps the ``quarkus-mongodb-client``
-extension and the {+driver-short+}. After installing ``quarkus-mongodb-panache``, you
-can create compact Panache entities to represent MongoDB documents. Several annotations,
-including ``@MongoEntity``, ``@BsonProperty``, and ``@BsonIgnore``, allow for
-customized mapping between entities and MongoDB documents.
+Use the MongoDB client extension to access the following features:
```

Comment:
Suggestion:

I think it could be helpful to stay consistent in each section with whether imperative mood (either using it or not using it) is used when presenting the features in each section. I think either way (Spring Data "You can use..." or Quarkus "Use the ...") works.